src/main/typescript/constants/apiConstants.ts

```diff
@@ -23,4 +23,9 @@ const createURL = (path: string) => `${baseURL}/${path}`
 export const listDepositsURL = createURL("deposit")
 export const deleteDepositURL = (id: DepositId) => createURL(`deposit/${id}`)
 
+export const loginURL  = createURL("auth/login204")
```

Comment:
don't forget to change this to `auth/login` (also in the server); otherwise integration with easy-deposit-api will fail